.changelog/22915.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:note
```

Comment:
should this one appear in the breaking changes list? 
```suggestion
```release-note:breaking-change
```